### PR TITLE
Creditor groups api

### PR DESCRIPTION
**Pull request by lomamech**
Adding API for Creditor Group and integration Test
@jniles  @DedrickEnc

### CODE REVIEW

File: server/controllers/finance/creditorGroups.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+  'use strict';
+
+  var sql;
+
+  sql =
+    'UPDATE creditor_group SET ? WHERE uuid = ?;';
+
+  db.exec(sql, [req.body, req.params.uuid])
+  .then(function () {
+    var uuid = req.params.uuid;
```
Comment (jniles) [inline server/controllers/finance/creditorGroups.js]: Why did you alias `req.params.uuid` here?

File: server/test/api/creditorGroups.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      })
+      .catch(helpers.handler);
+  });
+
+  it('GET /CREDITOR_GROUPS/:UUID returns a single Creditor Group ', function () {
+    return agent.get('/creditor_groups/' + creditorGroup.uuid)
+      .then(function (res) {
+        expect(res).to.have.status(200);
+        expect(res).to.be.json;
+        expect(res.body.uuid).to.equal(creditorGroup.uuid);
```
Comment (jniles) [inline server/test/api/creditorGroups.js:75]: Very nice.

**Comment by lomamech** [comment]
Improve Code 
@jniles

**Comment by jniles** [comment]
LGTM.

**Pull request merged**